### PR TITLE
Adds Butler Tutorial Notebook

### PR DESCRIPTION
**Pull request by djperrefort**
This PR adds a tutorial notebook on usage of the `Broker` package to the *Basics* directory.

### CODE REVIEW

File: Basics/README.rst [hunk 1]
```diff
@@ -57,3 +57,18 @@ This set of tutorial notebooks will help you explore the basic properties of the
           </a>
 
      - `Phil Marshall <https://github.com/LSSTScienceCollaborations/StackClub/issues/new?body=@drphilmarshall>`_
+     
+     * - **Broker Tutorial**
```
Comment (stargaser) [inline Basics/README.rst]: A better name is "Butler Tutorial". According to the [DM Glossary](https://confluence.lsstcorp.org/display/LKB/DM+Glossary#/), a broker is a "software system that ingests the Alert Stream and processes that information (e.g., filtering, cross-matching, categorizing). Outputs may include: processed streams of value-added Alerts, sorted lists."

File: Basics/README.rst [hunk 2]
```diff
@@ -57,3 +57,18 @@ This set of tutorial notebooks will help you explore the basic properties of the
...
+     - Demonstrates basic data access and manipulations using the data Butler
+     - `ipynb <https://github.com/LSSTScienceCollaborations/StackClub/blob/master/Basics/BrokerTutorial.ipynb>`__,
```
Comment (stargaser) [inline Basics/README.rst]: I recommend to rename the notebook to "ButlerTutorial.ipynb".

**Comment by djperrefort** [comment]
@stargaser the title is a typo - my brain was in multitasking mode and thinking about a different project 😆 . I'll fix the title.

**Comment by kadrlica** [comment]
@djperrefort thanks for the notebook! I notice that you are point to a dataset located at:
```
/global/cscratch1/sd/rearmstr/new_templates/diffim_template
```
I think this is on NERSC and is not accessible via the LSP at NCSA. Can you transfer the data to:
```
/project/shared/data/
```
on the LSP?

**Comment by djperrefort** [comment]
@kadrlica I've updated the text to match our recent hack session. I'm still getting a redirect error even when validating my ID with NCSA. Can you do a quick verification that everything runs start to finish for you? If so I think this can be merged in.

**Pull request merged**